internal/cli/helpers.go

```diff
@@ -0,0 +1,36 @@
+package cli
```

Comment:
We generally try to stay away from file names like `helpers` or `utils`. I would simply call this file for what it is: `internal/cli/url_scheme.go`